### PR TITLE
Dealreplication bugfix3

### PR DESCRIPTION
**Pull request by kernelogic**
Fix according to https://github.com/tech-greedy/singularity/pull/141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,16 +316,25 @@ Arguments:
...
   -r, --verified <verified>                            Whether to propose deal as verified. true|false. (default: "true")
   -s, --start-delay <startdelay>                       Deal start delay in days. (StartEpoch) (default: "7")
   -d, --duration <duration>                            Duration in days for deal length. (default: "525")
   -o, --offline <offline>                              Propose as offline deal. (default: "true")
-  -m, --max-deals <maxdeals>                           Max number of deals in this replication request per SP, per cron triggered. (default: "0")
-  -c, --cron-schedule <cronschedule>                   Optional cron to send deals at interval. Use double quote to wrap the format containing spaces.
-  -x, --cron-max-deals <cronmaxdeals>                  When cron schedule specified, limit the total number of deals across entire cron, per SP.
-  -xp, --cron-max-pending-deals <cronmaxpendingdeals>  When cron schedule specified, limit the total number of pending deals determined by dealtracking service, per SP.
+  -m, --max-deals <maxdeals>                           Max number of deals in this replication request per SP, per cron triggered.
```
Comment (xinaxu) [inline README.md:326]: try not make those line breaks. You can increase the terminal width to get the description shown in a single line

Comment (kernelogic) [reply README.md:326]: Does our lint limits line length?

Comment (xinaxu) [reply README.md:326]: Not sure, you can try

**Review by xinaxu — COMMENTED** [review]
Looks good overall

File: README.md (hunk 1 above)
Comment (xinaxu) [reply README.md:326]: Actually it does not

**Comment by kernelogic** [comment]
> Looks good overall

Can you approve it so I can merge?

**Review by xinaxu — APPROVED** [review]

**Pull request merged**